3rdparty/discord-rpc/CMakeLists.txt

```diff
@@ -0,0 +1,34 @@
+# discord_rpc
+add_library(3rdparty_discord-rpc INTERFACE)
```

Comment:
Probably a good idea to be semi-consistent with upstream Discord RPC's official CMakeLists.txt https://github.com/discord/discord-rpc/blob/master/CMakeLists.txt
And change this to DiscordRPC

Reply:
sure :) should i append this as a commit to this pr?

Reply:
Yeah, sounds good to me